jax/_src/numpy/lax_numpy.py

```diff
@@ -2558,9 +2586,38 @@ def _round_float(x: ArrayLike) -> Array:
 round_ = round
 
 
-@util.implements(np.fix, skip_params=['out'])
 @jit
 def fix(x: ArrayLike, out: None = None) -> Array:
+  """Round input to the nearest integer towards zero.
```

Comment:
This is a much better description than the one in `trunc` – I read it and I know immediately what the function does!